defaults/defaults.go

```diff
@@ -91,11 +91,7 @@ const (
 
 	IngressClassName = "cilium"
 
-	// HelmReleaseName is the default Helm release name for Cilium.
-	HelmReleaseName               = "cilium"
-	HelmValuesSecretName          = "cilium-cli-helm-values"
-	HelmValuesSecretKeyName       = "io.cilium.cilium-cli"
-	HelmChartVersionSecretKeyName = "io.cilium.chart-version"
+	HelmValuesSecretName = "cilium-cli-helm-values"
```

Comment:
These were unused, so I removed them along with the `HelmReleaseName`:
```go
	HelmValuesSecretKeyName       = "io.cilium.cilium-cli"
	HelmChartVersionSecretKeyName = "io.cilium.chart-version"
```

If that's not recommended, I can revert this commit.

Reply:
It's OK to remove them in a separate commit (as you did) as part of this PR. Thanks.